src/modules/chat_commands/store.js

```diff
@@ -19,17 +19,17 @@ class CommandStore {
     if (twitchCommandStore == null || this.commands.length === 0) {
       return;
     }
-    this.commands.forEach((command) => {
+    for (const command of this.commands) {
       twitchCommandStore.addCommand(command);
-    });
+    }
   }
 
   registerCommand(command) {
+    this.commands.push(command);
```

Comment:
is there ever a case where we might have duplicate commands registered?

Reply:
Nope, they replace the command if it already exists